test/integration/local/payment-request/update-manual-ledger-with-debt-data.test.js

```diff
@@ -0,0 +1,223 @@
+const db = require('../../../../app/data')
+const { updateManualLedgerWithDebtData, attachDebtToManualLedger } = require('../../../../app/manual-ledger')
+const { AR, AP } = require('../../../../app/processing/ledger/ledgers')
+const { PENDING, PASSED } = require('../../../../app/quality-check/statuses')
+
+let scheme
+let paymentRequest
+let qualityCheck
+let manualLedgerPaymentRequest
+
+const resetData = async () => {
+  await db.qualityCheck.truncate({ cascade: true })
+  await db.debtData.truncate({ cascade: true })
+  await db.scheme.truncate({ cascade: true })
+  await db.manualLedgerPaymentRequest.truncate({ cascade: true })
+  await db.paymentRequest.truncate({ cascade: true, restartIdentity: true })
+}
+
+describe('process payment requests', () => {
+  beforeEach(async () => {
+    await resetData()
+
+    scheme = {
+      schemeId: 1,
+      name: 'SFI'

```

Comment:
We have constant module for these -- you've used them in `test/integration/local/payment-request/update-payment-request-category.test.js`